src/main/java/net/minecraftforge/client/event/EntityViewRenderEvent.java

```diff
@@ -57,10 +61,13 @@ protected FogEvent(FogMode mode, Camera camera, double partialTick)
     }
 
     /**
+     * MODDERS: use RenderFogEvent
+     *
      * Event that allows any feature to customize the fog density the player sees.
      * NOTE: In order to make this event have an effect, you must cancel the event
      */
     @Cancelable
+    @Deprecated //TODO: remove in 1.19
```

Comment:
This should be a `@Deprecated(forRemoval = true, since = "1.18.2")` annotation and a javadoc with a `@deprecated` tag explaining that it will be removed in 1.19, why it is terminally deprecated, and linking to or showing alternatives to the event. (And to clarify, the `MODDERS:` part can be removed in favor of the `@deprecated` javadoc tag.)